proto/dymension/denommetadata/tokenmetadata.proto

```diff
@@ -0,0 +1,53 @@
+syntax = "proto3";
+package dymensionxyz.dymension.denommetadata;
```

Comment:
why is this file needed? you have this in the cosmos sdk already (proto/cosmos/bank/v1beta1/bank.proto)

Reply:
removed